server/src/ClientToDBAPI/RulerRoute/mainRoutes.ts

```diff
@@ -46,86 +47,16 @@ rulerRouter.post('/ruler/', UseCache, (request: Request, response: Response) =>
     }
     const parameters = <JSON> params
     rulerLogger.info(launchingAPIRequestLog(parameters), Severity.LOW);
-    console.log(callRullerApi(parameters))
     
-        // .then((result: any) => {
-        //     const data = result.data;
-        //     rulerLogger.info(validAPIResponseLog, Severity.LOW);
-        //     setToCache(request.originalUrl, data);
-        //     response.send(data);
-        // })
-        // .catch((error: string) => {
-        //     rulerLogger.error(invalidAPIResponseLog(error), Severity.HIGH);
-        //     response.send(error);
-        // });
+    axios.post(rulerApiUrl, parameters).then((data) => {
+        rulerLogger.info(validAPIResponseLog, Severity.LOW);
+        setToCache(req.originalUrl, data.data);
+        res.send(data.data);
+    }
+    ).catch((err: any) => {
+        rulerLogger.error(invalidAPIResponseLog(err), Severity.HIGH);
+        res.send(err);
```

Comment:
add .status(errorStatusCode).send(error);

Reply:
errorStatusCode is part of GraphqlHTTPRequest
I'll look for a way to get it here

Reply:
oops it's just means 500

Reply:
haha yes I was also surprised from this